src/builders.ts

```diff
@@ -132,6 +133,26 @@ export function broadcastTransaction(transaction: StacksTransaction, network: St
   });
 }
 
+export async function getAbi(
```

Comment:
Can we have some documentation for this like we do for the other builder functions?